app.py

```diff
@@ -693,17 +683,16 @@ def space_eu(level, step):
     g.lang = requested_lang()
     g.prefix = '/space_eu'
 
-    adventure_assignments = load_adventure_assignments_per_level(g.lang, level)
+    adventures = load_adventures_per_level(g.lang, level)
 
-    return hedyweb.render_assignment_editor(
+    return hedyweb.render_code_editor_with_tabs(
         request=request,
-        course=SPACE_EU_COURSE[g.lang],
+        course=HEDY_COURSE[g.lang],
```

Comment:
What is the goal for the `space_eu` course? The PR description said it would redirect to the regular Hedy course. If you want to do that,  you could replace this whole block with something like:

```py
    return redirect(url_for('index', level=level, step=step), code=302)
```

It would cut down on unnecessary code if we did that.